pages/conference/keynotes.vue

```diff
@@ -1,120 +1,133 @@
-                        <div class="flex pt-4">
-                            <div class="keynote__extLink">
-                                <ext-link
-                                    v-if="!!keynote.social_item.github.length"
-                                    :href="keynote.social_item.github"
-                                >
-                                    <fa
-                                        :icon="['fab', 'github-square']"
-                                        aria-hidden="true"
-                                        class="mr-2"
-                                    ></fa>
-                                </ext-link>
-                            </div>
-                            <div class="keynote__extLink">
-                                <ext-link
-                                    v-if="!!keynote.social_item.linkedin.length"
-                                    :href="keynote.social_item.linkedin"
-                                >
-                                    <fa
-                                        :icon="['fab', 'linkedin']"
-                                        aria-hidden="true"
-                                        class="mr-2"
-                                    ></fa>
-                                </ext-link>
-                            </div>
-                            <div class="keynote__extLink">
-                                <ext-link
-                                    v-if="!!keynote.social_item.twitter.length"
-                                    :href="keynote.social_item.twitter"
-                                >
+    <div class="py-10 md:py-24 lg:py24">
+        <banner>
+            <template #default>
+                <div
+                    class="hidden md:block md:w-full md:h-full"
+                    :style="bannerStyle"
+                ></div>
+            </template>
+            <template #text>
+                <core-h1 :title="$t('title')"></core-h1>
+                <p class="w-full md:w-3/4 lg:w-3/4 leading-6">
+                    {{ $t('intro') }}
+                </p>
+            </template>
+        </banner>
+        <i18n-page-wrapper>
+            <article
+                v-for="keynote in keynotesData"
+                :id="getKeynoteId(keynote)"
+                :key="keynote.id"
+                class="keynote"
+            >
+                <div class="keynote__photo">
+                    <img
+                        :src="keynote.speaker.photo"
+                        :alt="keynote.speaker[$makeKey(locale, 'name')]"
+                    />
+                </div>
+                <div class="keynote__name">
+                    {{ keynote.speaker[$makeKey(locale, 'name')] }}
+                </div>
+                <div class="keynote__links">
+                    <div class="keynote__extLink">
+                        <ext-link
+                            v-if="!!keynote.social_item.facebook.length"
```

Comment:
**suggestion (non-blocking):** I guess we should move `v-if` into the `keynote__extLink` div since it will take some space when there's no `ext-Link` in it